indico/modules/events/editing/client/js/components/FileManager/index.jsx

```diff
@@ -9,8 +9,11 @@ import _ from 'lodash';
 import React, {useCallback, useReducer, useContext, useRef, useMemo} from 'react';
 import PropTypes from 'prop-types';
 import {useDropzone} from 'react-dropzone';
-import {Icon} from 'semantic-ui-react';
-import {FileManagerContext, filePropTypes, uploadFiles, deleteFile, mapFileTypes} from './util';
+import {Button, Icon} from 'semantic-ui-react';
+
+import {Param, Translate} from 'indico/react/i18n';
+
```

Comment:
👾

Reply:
That was intentional - to separate our general modules from the local ones (local to the component path)